README.md

```diff
@@ -1 +1,21 @@
-# wcc-backend
\ No newline at end of file
+# Women Coding Community Platform Backend Service
+
+## Locally Development
```

Comment:
Can add the endpoint of the localhost - ( Anyone new might not know)
eg - http://localhost:8080/api/cms/v1/team